src/modules/page-anchor/page-anchor.component.spec.ts

```diff
@@ -53,10 +53,10 @@ describe('StachePageAnchorComponent', () => {
     expect(id).toBe('test-content');
   });
 
-  it('should call the addHashToUrl method when the heading is clicked', () => {
+  it('should call the addHashToUrl method when the icon is clicked', () => {
```

Comment:
No more clicking on the header itself to scroll to it? I suppose that works too.